### PR TITLE
feat(server): implement /service route

### PR DESCRIPTION
**Pull request by guyts96**

### CODE REVIEW

File: server/src/services/serviceService.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                return;
+            }
+            Object.keys(haServices[domain]).forEach((service) => {
+                services.push({
+                    name: `${domain}.${service}`,
+                    fields: Object.keys(haServices[domain][service].fields),
+                });
+            });
+        });
+        return services;
```
Comment (asoran) [inline server/src/services/serviceService.ts:23]:
```suggestion
    return Object.keys(haServices)
        .filter(domain => domains.indexOf(domain) === -1)
        .flatMap(domain => {
            return Object.keys(haServices[domain]).map(service => {
                name: `${domain}.${service}`,
                fields: Object.keys(haServices[domain][service].fields),
            });
        });
```

**Review by asoran — APPROVED** [review]
:zap:

File: server/src/services/serviceService.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import socketForwarder from '../forwarders/socketForwarder';
+import { HaService } from '../types/haTypes';
+import { Service } from '../types/service';
+import entityService from './entityService';
+
+class ServiceService {
```
Comment (Lukyio) [inline server/src/services/serviceService.ts:6]: 🤣

**Review by Lukyio — APPROVED** [review]
♌

**Pull request merged**